reana_client/config.py

```diff
@@ -50,3 +50,6 @@
     "queued",
 ]
 """Available run statuses."""
+
+WARNING_TAGS = ["latest", "master"]
+"""Warns user if above tags are used."""
```

Comment:
Please scope this variable more, since the "tags" concept could in theory refer to Git tags. For example `VALIDATOR_ENVIRONMENT_IMAGE_SUSPECTED_TAGS` or some such.